config/pdb/pdb.yaml

```diff
@@ -3,7 +3,6 @@ kind: PodDisruptionBudget
 metadata:
   name: controller-manager
 spec:
-  minAvailable: 1
```

Comment:
please revert

Reply:
I’m not sure if there is a way to preserve this for deploy/gatekeeper.yaml while enabling this new conditional to support minUnavailable and maxUnavailable in pdb. Do you @sozercan ?